File: problem-set-1/mario-more/mario.c
#include <stdio.h>

void bricksbuilder(int height){
  int i, j;
  for (i = 1; i <= height; i++)
      for (j = 1; j < height - i; j++)
          printf(" ");
      for (j = 1; j <= i; j++)
          printf("#");
      printf("  ");
      for (j = 1; j <= i; j++)
          printf("#");
      printf("\n");
}
int main(void){
  int height;
  printf("Height: ");
  scanf("%d", &height);
  if (height < 1 || height > 8)
    printf("Height must be between 1 and 8!\n");
  else
    bricksbuilder(height);
}

File: problem-sets/problem-set-06/sentimental-mario-more/mario.py
while (True):
  try:
    height = int(input("Height: "))
    if 1 <= height <= 8:
     break
    else:
    print("Height must be between 1 and 8")
  except ValueError:
    print("Please enter a valid integer")

for row in range(1, height + 1):
  spaces = " " * (height - row)
  hashes = "#" * row
  print(f"{spaces}{hashes}  {hashes}")